macapp/App/Sources/App/AdminWindow/AdminWindowFeature.swift

```diff
@@ -278,6 +278,7 @@ extension AdminWindowFeature.RootReducer {
 
       case .webview(.healthCheck(.fixNotificationPermissionClicked)):
         return .exec { _ in
+          await self.device.requestNotificationAuthorization()
```

Comment:
this is essentially the only important line of the whole PR, the rest is a test and bumping version stuff. if we've already requested auth, this does nothing, but if not, it's what makes the button actually work.